quote.c

```diff
@@ -47,7 +47,16 @@ void sq_quote_buf_pretty(struct strbuf *dst, const char *src)
 {
 	static const char ok_punct[] = "+,-./:=@_^";
 	const char *p;
-
+	
```

Comment:
this newline makes me think that you've been writing with CRLFs. `git log --check` will help find out.